packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.ts

```diff
@@ -170,6 +172,8 @@ describe("End to end chunked encoding", () => {
 
 		// This function is declared in the test to have access to the original uniform chunk for comparison.
 		function stringifier(content: unknown) {
+			// TODO: use something other than `any`
```

Comment:
Do you think we should tag all of these "TODO"s with a work item? We could also make a tech debt item later that says "search for the string 'use something other than `any`'", but a work item tag seems a little more robust.

Reply:
Up to you. My main goal is to prevent new instances. However you all want to track addressing the rest is up to you 😜

Reply:
You can just throw it under unparented and someone will move it to the right place
https://dev.azure.com/fluidframework/internal/_sprints/taskboard/Team%20Taylor/internal/Team%20Taylor%20-%202024.Frappuccino

Thank you :)

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/8033